liblsp/README.md

```diff
@@ -0,0 +1,19 @@
+# LSP C++ Library
```

Comment:
Can you state which version of the LSP protocol we are targeting? And also link to the correct version of the spec: https://microsoft.github.io/language-server-protocol/specifications/specification-current/

Reply:
Was this added? I couldnt find it.

Reply:
> 
> 
> Was this added? I couldnt find it.

https://github.com/ethereum/solidity/pull/8125/files#diff-0275e6003037b7c2499927f67d7018345bcb714fcd612f1bbdb32a11a1621108R8

I hope that is the place you meant it to be put. I can also add that as the base class header documentation of the protocol implementation.